src/pattern_library/pattern_colorloop_sin.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "../pattern_internals/pattern.h"
+
+class Pattern_Colorloop_Sin: public Pattern {
```

Comment:
There may be a more descriptive name for this, but that's true for a lot of patterns. Maybe that'd be better handled with comment documentation? Pretty soon I'll comment the existing patterns and start requiring that so this could be part of that change.